.codeclimate.yml

```diff
@@ -31,6 +26,11 @@ checks:
     # non-JSX code, but it doesn't seem possible to change
     # the threshold on a per-language basis here.
     enabled: false
+  argument-count:
+    # This isn't a very useful check for Python code because
+    # it's perfectly fine to have functions that take lots
+    # of arguments, as long as most of them are passed as kwargs.
+    enabled: false
```

Comment:
@samaratrilling just curious, why the change in the order of these configurables?